src/bootstrap/src/utils/exec.rs

```diff
@@ -286,3 +287,15 @@ impl From<Output> for CommandOutput {
         }
     }
 }
+
+/// If in a CI environment, forces the command to run with colors.
+pub fn force_coloring_in_ci(ci_env: CiEnv, cmd: &mut BootstrapCommand) {
+    if ci_env != CiEnv::None {
+        // Due to use of stamp/docker, the output stream of bootstrap is not
+        // a TTY in CI, so coloring is by-default turned off.
+        // The explicit `TERM=xterm` environment is needed for
+        // `--color always` to actually work. This env var was lost when
+        // compiling through the Makefile. Very strange.
+        cmd.env("TERM", "xterm").args(["--color", "always"]);
+    }
+}
```

Comment:
What about moving this under `impl BootstrapCommand` ?

Reply:
Good idea, done that.